releases/release-1.24/release-notes/release-notes-draft.json

```diff
@@ -44,6 +63,33 @@
     "duplicate": true,
     "duplicate_kind": true
   },
+  "103061": {
+    "commit": "5211a4b2147ce745649e77687a87cdebc08a6707",
+    "text": "The node.k8s.io/v1alpha1 RuntimeClass API is no longer served. Use the node.k8s.io/v1 API version, available since v1.20",
+    "markdown": "The node.k8s.io/v1alpha1 RuntimeClass API is no longer served. Use the node.k8s.io/v1 API version, available since v1.20 ([#103061](https://github.com/kubernetes/kubernetes/pull/103061), [@SergeyKanzhelev](https://github.com/SergeyKanzhelev)) [SIG API Machinery, CLI, Node and Testing]",
```

Comment:
```suggestion
    "markdown": "The `node.k8s.io/v1alpha1 `RuntimeClass API is no longer served. Use the `node.k8s.io/v1` API version, available since v1.20 ([#103061](https://github.com/kubernetes/kubernetes/pull/103061), [@SergeyKanzhelev](https://github.com/SergeyKanzhelev)) [SIG API Machinery, CLI, Node and Testing]",
```